website/content/specification/syntax/constraints.md

```diff
@@ -14,27 +14,25 @@ TODO: P3: Address issue https://github.com/usnistgov/metaschema/issues/325
 
 ## Enumerated values
 
-Additionally, flags may be constrained to a set of known values listed in advance.
+One kind of Metaschema constraint is restricting assembly, field, or flag value(s) with `allowed-values` enumerations. Metaschema processors MUST process `allowed-values` enumerations as defined with the `allow-other` attribute.
 
-This restriction can be either:
+1. When the `allow-other` attribute is defined as `allow-other="no"`, the processor MUST strictly validate content instances with enumerations: only defined `enum` values are valid for the given target(s).
+2. When the `allow-other` attribute is defined as `allow-other="yes"`, the processor MUST loosely validate content instances with enumerations: both `enum` values and other values are valid.
```

Comment:
Need to qualify "loosely" as it is not clear what this means. This could be a SHOULD or MAY.

Reply:
```suggestion
2. When the `allow-other` attribute is defined as `allow-other="yes"`, the processor MAY loosely validate content instances with enumerations. A validation warning MAY be raised for any value that does not match an enumerated value.
```